### PR TITLE
Clean up `pom.xml` and `Dockerfile`

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
     <version>0.0.17</version>
   </parent>
+
+  <artifactId>scalecube-organization-parent</artifactId>
+  <version>2.0.14-SNAPSHOT</version>
+
   <packaging>pom</packaging>
 
-  <name>scalecube-organization-parent</name>
+  <name>ScaleCube Organization Service</name>
```
Comment (aharonha) [inline pom.xml]: this is not the service itself. please consider another name

File: pom.xml [hunk 2]
```diff
@@ -23,10 +26,11 @@
   </scm>
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (aharonha) [inline pom.xml]: don't we set it on parent pom?

File: scalecube-organization/pom.xml [hunk 3]
```diff
@@ -11,11 +13,10 @@
   <packaging>jar</packaging>
 
   <name>scalecube-organization</name>
-  <url>http://maven.apache.org</url>
 
   <properties>
-    <serviceName>${project.artifactId}</serviceName>
     <mainClass>io.scalecube.server.OrganizationServiceRunner</mainClass>
+    <extratags>${env.TRAVIS_COMMIT}</extratags>
```
Comment (aharonha) [inline scalecube-organization/pom.xml]: did you test it? does it deploy each commit to docker hub? is it what you want?

File: pom.xml (hunk 1 above)
Comment (dmytro-lazebnyi) [reply pom.xml]: Just aligned with a similar name in `scalecube-configuration-service`

https://github.com/scalecube/scalecube-configuration-service/blob/9b03721dd77f92a90df831e65200f171f5ef7d5f/pom.xml#L15

File: pom.xml (hunk 2 above)
Comment (dmytro-lazebnyi) [reply pom.xml]: Yep, good point. Removed from `scalecube-organization-service` pom.

File: scalecube-organization/pom.xml (hunk 3 above)
Comment (dmytro-lazebnyi) [reply scalecube-organization/pom.xml]: > is it what you want?

It is common practice for scalecube services, isn't it?

Comment (aharonha) [reply scalecube-organization/pom.xml]: if it is required from a service to be tested as a container across multiple versions (mostly BM testing) we are using it. ask @artem-v

**Pull request merged**